src/i18n/en.js

```diff
@@ -27,6 +27,19 @@ const urlSiteContact = `${urlSite}/en/contact/`;
 const urlControlPanelMain = `${urlControlPanel}/ControlPanel`;
 
 export default {
+  campaign_history: {
+    status: {
+      active: 'Active',
+      inactive: 'Active not associated with lists',
```

Comment:
```suggestion
      inactive: 'Active not associated with Lists',
```

Reply:
In English the states with uppercased letters are as follows:
Active
Active not Associated to List
On Hold waiting for Upgrade
Removed by Client
Removed by Hard-Bounced
Removed by Never-Opened
Removed by Soft-Bounced
Removed by Subscriber

Reply:
fixed